tests/functional/object/mcg/test_store_status_transitions.py

```
@@ -0,0 +1,198 @@
+        )
+        aws_client = cld_mgr.aws_client
+        logger.info(
+            f"Created {store_type} {store.name} backed by "
+            f"AWS bucket {store.uls_name}"
+        )
+
+        # 2. Verify the store is in Ready phase and OPTIMAL mode
+        assert_func(store.name, constants.STATUS_READY, BS_OPTIMAL, mcg_obj)
+
+        # 3. Apply a Deny bucket policy to block access
+        logger.info(f"Blocking access to underlying bucket {store.uls_name}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)